### ISSUE TITLE
[Dungeon] Heroic Old Hillsbrad foothills reset

### ISSUE THREAD

**Issue by NatsumeLily** [Atlantiss/NetherwingBugtracker#6347]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: Heroic old hillsbrad foothils.

**Current behaviour**: After completing the instance, when the dungeon soft resets Thrall and Taretha will reset. Allowing you to start the event again and kill the last boss again. This can be done for every soft reset (or sooner, didn't check), allowing you to kill the last boss every 30mins.

**Expected behaviour**: (...Need I add this?) Shouldn't be able to kill a heroic last boss multiple times a day.

**Server Revision**: 3157

**Comment by Wolffenstein** [comment]
Cannot reproduce, I did the entire event, soft resetted the dungeon and Thrall is nowhere to be found (nor is Taretha). Considering the time of this bug report, are you sure it didn't just coincide with the daily reset?

**Comment by NatsumeLily** [comment]
So the full story.
I did the daily with 2 guildmates but we had to stop before killing the 2nd boss because of a troll, I logged out and went to bed. When I logged back in about 12 hours later I checked the instance and found the last boss available (thinking someone did it but couldn't kill last boss), so I got a group together and killed him.
I later asked the guildmate if they finished it and they did, so the last boss was killed twice in the ID. I then checked to see if it was bugged 1 hour before reset (raid info said the reset was 1 hour away), and thrall and taretha had reset again. Allowing me to talk to her and start the event again.
